client/src/components/Login/index.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import './index.css';
```

Comment:
This should be removed or an index.css file should be created

Reply:
The index.css file is in the src folder.